IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo

```diff
@@ -228,16 +230,19 @@ block ReaderTMY3 "Reader for TMY3 weather data"
 protected
   final parameter String absFilNam = IBPSA.BoundaryConditions.WeatherData.BaseClasses.getAbsolutePath(filNam)
     "Absolute path of the file";
+  // For evalating time span of weather data
```

Comment:
Why does `timeSpan` need a start value if it is assigned anyway?
Also, this should probably be `final`

Reply:
Since the fileName is "", even if the parameters are assigned no values are outputed by the function and the equation blocks relating to the values of timeSpan are considered invalid. My solution to this was to add start values, but I am open to any ideas which are better.
Made the parameters final.

Reply:
I ended up removing the start values as they started acting crazy in the example. Anyway, in order to have definite values, I gave a value to parameter filName in ReaderTMY3. The examples should work now.

Reply:
There is a typo: evalating instead of evaluating